ui/config/config.js

```diff
@@ -393,6 +394,22 @@ export default class CactbotConfigurator {
     parent.appendChild(div);
   }
 
+  buildUrl(parent, opt, group) {
```

Comment:
I'd like another set of eyes on this, I've attempted to mimic the buildFloat code and from some basic testing it appears to work but I'm not really familiar with the underlying system here.

Reply:
I haven't tested it locally, but this looks fine to my eyes, thanks!